### PR TITLE
update gallery for `UnicodePlots`

### PR DESCRIPTION
**Pull request by t-bltg**
Following https://github.com/JuliaPlots/Plots.jl/pull/4149, and https://github.com/JuliaPlots/UnicodePlots.jl/pull/239.

### CODE REVIEW

**Comment by johnnychen94** [comment]
It looks like the new release v1.27.2 does not trigger the documentation rebuild.

**Comment by t-bltg** [comment]
I have to merge https://github.com/JuliaPlots/UnicodePlots.jl/pull/240 for this PR to work.

**Comment by johnnychen94** [comment]
Sorry, I meant https://docs.juliaplots.org/stable/ is still targeting v1.27.1 at the moment and I didn't see docs CI run against `v1.27.2` label in https://github.com/JuliaPlots/Plots.jl/actions/workflows/docs.yml

**Comment by BeastyBlacksmith** [comment]
> So any commit to master will now build the docs for `1.27.2`.

Without tag they only build for `dev`. You can trigger the build manually on the actions tab (which I did now) or you don't put `[skip ci]` on the commit massage that gets the tag.

**Comment by johnnychen94** [comment]
I see you need https://github.com/johnnychen94/DemoCards.jl/pull/109 for debug purposes. How about committing `docs/Manifest.toml` file with the dev-ed version of that branch via `git add -f docs/Manifest.toml`? When everything works fine we can safely delete the Manifest file.

For local development, you can first run `generate_cards(:unicodeplots)` to generate the files, and then use `DemoCards.preview_demos("docs/src/gallery/unicodeplots")` for fast preview.

**Comment by t-bltg** [comment]
> I see you need https://github.com/johnnychen94/DemoCards.jl/pull/109 for debug purposes. How about committing docs/Manifest.toml file with the dev-ed version of that branch via git add -f docs/Manifest.toml? When everything works fine we can safely delete the Manifest file.

I don't understand this. I work locally and I don't any Manifest file here.

> For local development

I am sticking with the doc.yml command: `JULIA_DEBUG=Documenter,DemoCards GITLAB_CI=true GKSwstype=nul julia --project=docs/ docs/make.jl`

**Comment by johnnychen94** [comment]
```julia
(docs) pkg> add https://github.com/t-bltg/DemoCards.jl.git#debug
     Cloning git-repo `https://github.com/t-bltg/DemoCards.jl.git`
    Updating git-repo `https://github.com/t-bltg/DemoCards.jl.git`
    Updating registry at `~/.julia/registries/General.toml`
...

shell> git add docs/Manifest.toml -f

shell> git status
Refresh index: 100% (249/249), done.
On branch master
Your branch is up to date with 'origin/master'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        new file:   docs/Manifest.toml
...

shell> git commit -m "debug: temporarly commit docs/Manifest.toml"
[master c22be33] debug: temporarly commit docs/Manifest.toml
 1 file changed, 645 insertions(+)
 create mode 100644 docs/Manifest.toml
```

File: docs/gallery/unicodeplots/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# UnicodePlots
+
+To switch to UnicodePlots backend, you can use:
```
Comment (waldyrious) [inline docs/gallery/unicodeplots/index.md:3]: Just a tiny nit :smile:

```suggestion
To switch to the UnicodePlots backend, you can use:
```

Comment (johnnychen94) [reply docs/gallery/unicodeplots/index.md:3]: Might also need to change for all other backends...

Comment (t-bltg) [reply docs/gallery/unicodeplots/index.md:3]: @waldyrious, thanks.
I'm currently struggling with `FreeTypeAbstraction` and rendering `png` files from `UnicodePlots` plots.
I will come back and update this PR later.

**Comment by t-bltg** [comment]
CI passes, let's merge this now and see how it renders.

**Pull request merged**